File: src/com/basmilius/ps/bastools/action/AboutAction.java
package com.basmilius.ps.bastools.action;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.application.ApplicationManager;
import com.intellij.openapi.ui.Messages;

public class AboutAction extends AnAction
{

	public AboutAction()
	{
		super("About Bas Tools");
	}

	@Override
	public final void actionPerformed (final AnActionEvent aae)
	{
		ApplicationManager.getApplication().invokeLater(() -> Messages.showMessageDialog(aae.getProject(), "Bas Tools version 1.0.0.", "Bas Tools", Messages.getInformationIcon()));
	}

}

File: src/com/basmilius/ps/bastools/action/AboutAction.kt
package com.basmilius.ps.bastools.action

import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.application.ApplicationManager
import com.intellij.openapi.ui.Messages

/**
 * Class AboutAction
 *
 * @author Bas Milius
 * @package com.basmilius.ps.bastools.action
 */
class AboutAction : AnAction("About Bas Tools")
{

	/**
	 * {@inheritdoc}
	 *
	 * @author Bas Milius
	 */
	override fun actionPerformed(aae : AnActionEvent)
	{
		ApplicationManager.getApplication().invokeLater { Messages.showMessageDialog(aae.project, "Bas Tools 1.0.4, go to bas-tools.atbm.nl for more information about this plugin.", "Bas Tools", Messages.getInformationIcon()) }
	}

}
